crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -280,12 +309,18 @@ It can greatly improve performance (and readability) in such situations as it pr
         // use timeline_name as part of id, so that egui stores different pan/zoom for different timelines
         let plot_id_src = ("plot", &timeline_name);
 
-        let zoom_both_axis = !ui.input(|i| i.modifiers.contains(controls::ASPECT_SCROLL_MODIFIER));
+        let locked_range = zoom_behavior.unwrap_or_default() == ZoomBehavior::LockToRange;
```

Comment:
nit: would be easier to read if this was `allow_non_uniform_zoom` or `unlocked_range`


```suggestion
        let allow_non_uniform_zoom = zoom_behavior.unwrap_or_default() != ZoomBehavior::LockToRange;
```

Reply:
finding it a bit odd though that the blueprint option disables the modifier key, thus invalidating what the doc says